.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 # keep this blank to make sure there are no before_install steps
 before_install:
-
+  npm install codecov
```

Comment:
Thx


Ari

On Mon, Apr 25, 2016 at 5:23 PM, OJ Kwon <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/shelljs/shx/pull/33#discussion_r61012761>:
>
> > @@ -6,7 +6,7 @@ env:
> >
> >  # keep this blank to make sure there are no before_install steps
> >  before_install:
> > -
> > +  npm install codecov
>
> For me either case is fine, since current way's my preferences. I'll
> update PR shortly.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/shelljs/shx/pull/33/files/3627c45292808e560b5a4b4dba6df50da9be666c#r61012761>
>